### DISCUSSION TITLE
Error when use settings.ffmpeg.content.copy.relaxed_compatibility_check.set(true) version 2.0.x

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#4012] [User Support]
Hi, the settings "settings.ffmpeg.content.copy.relaxed_compatibility_check.set(true)" are included from which version?

I'm using 2.0.7 (started using for audio and now need to use for video too)

```
At /home/autovj.liq, line 13, char 0-15:
settings.ffmpeg.content.copy.relaxed_compatibility_check.set(true)

Error 5: this value has no method content
```
I want to use this option to allow multiple video resolutions because I'm getting this error when the videos are not the same configuration:

`[decoder:2] Error while checking file's content: Frame_content.Incompatible_format: formats ffmpeg.audio.copy(sample_rate=44100,sample_format=fltp,channel_layout="stereo",codec="aac") and ffmpeg.audio.copy(sample_rate=22050,sample_format=fltp,channel_layout="stereo",codec="aac") are incompatible!
`

**Comment by gAlleb** [comment]
Hi @caf2022. As I recall `settings.ffmpeg.content.copy.relaxed_compatibility_check` has been introduced in `2.1.2` version. 

So you better upgrade. 

Any particular reason you are using outdated `2.0.7`?

`2.2.5` has new features and improvements and you can use `settings.ffmpeg.content.copy.relaxed_compatibility_check := true` there.

**Comment by caf2022** [comment]
Hi @gAlleb thanks for the help.

Have no special reason, I was using the liquidsoap since version 2.0.4 and works fine and as we say in my country, "when a team is winning, don't change anything" but now I will need to change :)
